GHzDACs/jump_table.py

```diff
@@ -10,8 +10,13 @@
 
 
 class JumpEntry(object):
-    """ A single entry in the jump table.
+    """A single entry in the jump table.
 
+    Attributes:
+        from_addr(int): SRAM cell at which this entry fires.
+        to_addr(int): SRAM cell to which we go after executing this
+            entry.
```

Comment:
Not sure what you mean by this. If `to_addr` is optional, shouldn't it default to `None` or something?